src/cosalib/build.py

```diff
@@ -164,9 +164,36 @@ def meta(self):
             self._build_json["meta"] = self.__get_json("meta")
         return self._build_json["meta"]
 
+    def refresh_meta(self):
```

Comment:
When do we need this? Feels like those callsites should go through updating the object and calling `meta_write()` instead?

Reply:
This is used in the cases of replicating the AWS images to make the operation idempotent. The `@retry` decorator calls the function up to three times until the replication completes. At the end of the replication, we write the `meta.json` which is the source of truth. If we publish to N regions but fail in 1, on retry, we refresh with the latest `meta.json` and carry on.